src/Authentication/Authentication/IAuthenticator.cs

```diff
@@ -0,0 +1,75 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using System;
+using System.Security;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Common.Authentication
+{
+    /// <summary>
+    /// Interface for authentication with distributed responsibility
+    /// </summary>
+    public interface IAuthenticator
```

Comment:
Is this the 'new' interface for auth?

Reply:
Yes

Reply:
It is the new interface until we replace AuthenticationFactory - this will wait for the update from ADAL -> MSAL